examples/react-nodejs/package.json

```
@@ -1,45 +1,29 @@
 {
   "private": true,
+  "type": "module",
+  "scripts": {
+    "dev": "concurrently \"node server.js\" \"vite\""
```

```suggestion
    "start:client": "vite",
    "start:server": "node server.js",
    "start": "concurrently \"node server.js\" \"vite\""
```

Could you also add a `build` script here for vite similar to `react-scripts`?